test/e2e/lib/pages/reader-page.js

```diff
@@ -26,6 +26,14 @@ export default class ReaderPage extends AsyncBaseContainer {
 		return URL.parse( href ).host;
 	}
 
+	async shareLatestPost() {
+		await driverHelper.clickWhenClickable( this.driver, by.css( '.reader-share__button' ) );
+		return await driverHelper.clickWhenClickable(
+			this.driver,
+			by.css( '.reader-popover .site__content' )
+		);
+	}
+
```

Comment:
In case if it happens, that e2e user doesn't follow any site and the reader is empty, this will fail. What do you think to put this in `try` block and throw an error?

Reply:
Don't users follow their own sites at least? We could post something as a step before this.

Reply:
> What do you think to put this in try block and throw an error?

Curious, is this to make the error more explicit? Or does the suite stay green and skip over remaining steps?

Reply:
Oh gotcha, for that makes sense for the page helper. I can update that and also bulletproof the suite a bit.

Reply:
Thanks, updated in 26d3c1b, if I explicitly make this fail we get a clearer error instead of a timeout:

<img width="585" alt="Screen Shot 2019-12-11 at 11 27 05 AM" src="https://user-images.githubusercontent.com/1270189/70653522-84695380-1c09-11ea-94bc-c61408fcdbb4.png">

Reply:
Thanks. This looks great!